framework/include/dirackernels/ConstantPointSource.h

```diff
@@ -19,22 +20,30 @@ template <>
 InputParameters validParams<ConstantPointSource>();
 
 /**
- * TOOD
+ * A ConstantPointSource DiracKernel is used to create constant point sources.
+ * Coordinates and values for the point source can be given in the input or read from file
  */
-class ConstantPointSource : public DiracKernel
+class ConstantPointSource : public DiracKernel, public VectorPostprocessorInterface
 {
 public:
   static InputParameters validParams();
-
+  /// Need initialSetup because vectorpostprocessor will not be available at construction
+  virtual void initialSetup() override;
+  /// map to associate points with their value in computeQpResidual
```

Comment:
these comments are probably better placed in the .C file as they do not appropriately document these member methods

Reply:
I reverted the vpp part of this so now everything is in the constructor and these comments were removed